svcdef/consolidate_http.go

```diff
@@ -25,7 +34,9 @@ func consolidateHTTP(sd *Svcdef, protoFiles []io.Reader) error {
 		lex := svcparse.NewSvcLexer(pfile)
 		protosvc, err := svcparse.ParseService(lex)
 		if err != nil {
-			if strings.Contains(err.Error(), "'option' or 'additional_bindings'") {
+			fmt.Println(err.Error())
```

Comment:
Do we want this to both print the error in this way and print out the warning? I'm fine with that, just want to make sure this is not for debugging.